.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
do we need to check this in? i'm worried it'll mess up my settings when I fetch this